Sources/PointFreeRouter/Routes.swift

```diff
@@ -174,12 +174,16 @@ let routers: [Router<Route>] = [
   .case { .collections(.section($0, $1)) }
     <¢> get %> "collections" %> pathParam(.tagged(.string)) <%> pathParam(.tagged(.string)) <% end,
 
+  parenthesize(.case { .collections(.episode($0, $1, $2)) })
+    <¢> get %> "collections"
+    %> pathParam(.tagged(.string))
+    <%> pathParam(.tagged(.string))
+    <%> pathParam(.episodeIdOrString)
+    <% end,
```

Comment:
this route produces things like `/collections/map-zip-flatmap/map/e13-many-faces-of-map`. 

down to workshop this url a bit more if you have ideas.

Reply:
I think it works well. Easy to manually chop off a path component to get to another valid page. Could be down to prepend the section with its sequence, though.